teradata/tox.ini

```diff
@@ -0,0 +1,39 @@
+[tox]
+isolated_build = true
+minversion = 2.0
+skip_missing_interpreters = true
+basepython = py38
+envlist = py38, py38-sandbox
+
+[testenv]
+ensure_default_envdir = true
+envdir =
+    py38: {toxworkdir}/py38
+description =
+    py38: e2e ready
+    py38-sandbox: e2e ready if TERADATA_SERVER
```

Comment:
Not too familiar with tox files, is `if TERADATA_SERVER` actual logic checking if `TERADATA_SERVER` is an environment variable?

Reply:
Yup, it's checking whether the `TERADATA_SERVER` environment variable is available locally. I set that condition so that the `py38-sandbox` environment doesn't run when running `ddev env test teradata -ne` if the env var isn't available. Since `py38-sandbox` is intended to be run locally, should I take the condition out and have the expectation that the environment variable will be set?

Reply:
I think the current way you have it is fine, since you also note it in the `test/README`.